pkg/service/capabilities/capabilities.go

```diff
@@ -27,5 +43,45 @@ func (s *service) Get() (*capabilities.GetResponse, error) {
 		res.Ingress = true
 	}
 
+	if ok, err := s.hasServiceMonitor(ctx); err != nil {
+		return nil, err
+	} else {
+		res.HasPrometheusServiceMonitor = ok
+	}
+
+	if ok, err := s.hasCustomMetricsAPI(ctx); err != nil {
+		return nil, err
+	} else {
+		res.HasCustomMetrics = ok
+	}
+
 	return res, nil
 }
+
+func (s *service) hasServiceMonitor(ctx context.Context) (bool, error) {
+	crds, err := s.clientSet.ApiextensionsV1().CustomResourceDefinitions().List(ctx, metav1.ListOptions{})
```

Comment:
Maybe just get by name instead of listing and traversing?

Reply:
Yea makes sense